validphys2/src/validphys/process_options.py

```diff
@@ -159,7 +160,8 @@ def _hqp_yq_xq2map(kin_dict):
 
 def _hqp_yqq_xq2map(kin_dict):
     # Compute x, Q2
-    ratio = np.sqrt(kin_dict[_Vars.m_t2]) / kin_dict[_Vars.sqrts]
+    mass2 = _get_or_fail(kin_dict, [_Vars.m_t2, _Vars.m_ttBar])
```

Comment:
The others will fail if they are not there (because it's a dict). It's just a way of having some general control over the situation of having more than one possibility for one variable. Although right now is basically a dictionary.get with a fallback